src/spm/spmutil.go

```diff
@@ -32,19 +31,13 @@ var (
 	hsmSlot          = flag.Int("hsm_slot", 0, "The HSM slot number; required")
 	genKG            = flag.Bool("gen_kg", false, "Generate KG; optional")
 	genKCA           = flag.Bool("gen_kca", false, "Generate KCA; optional")
-	loadHighSecKS    = flag.Bool("load_high_sec_ks", false, "Load high security KDF seed; optional")
```

Comment:
Should we just delete spmutil altogether? Instead of just removing functionality?

Reply:
Integration tests will fail if we have not switched over to hsmtool yet and you remove functionality from spmutil

Reply:
I still have a couple more functions to implement in `hsmtool` to be able to remove this. Will do it in a follow up change.